library.py

```diff
@@ -52,8 +54,11 @@ def remove_book(filename, isbn):
 
     # How can we *remove* an item from a dictionary?
     # Write code to delete the book keyed by isbn in the space below
-
+     if 'isbn' in remove_book:
```

Comment:
`remove_book` isn't defined within the scope of the function, it's the _name_ of the function.